pycoin/cmds/ku.py

```diff
@@ -43,7 +43,7 @@ def get_entropy():
     return entropy
 
 
-def create_output(item, key, output_key_set, subkey_path=None):
+def create_output(item, key, output_key_set, args, subkey_path=None):
```

Comment:
Instead of making it an option, can you simply look to see if the network supports bip49 and/or bip84, and always print it if so?

This will break a lot of tests because it will add output. Go ahead and eyeball the diffs, and if they look okay, I can quickly repair them after I commit your fix (using this `REPAIR_TESTS=1 py.test tests` trick – it will rewrite the broken tests. If you can't figure out what I'm talking about, I'm happy to repair the tests, since it takes just a minute).